config/application.rb

```diff
@@ -1,16 +1,19 @@
 # frozen_string_literal: true
 require_relative 'boot'
-
+require 'rails/all' 
```

Comment:
Why did you add this line?  This shouldn't be here for this app.

Reply:
it wouldn't run without it after adding the gems

Reply:
removed